### DISCUSSION TITLE
How do I set workers in model.predict

### DISCUSSION THREAD

**Discussion by aribornstein** [PyTorchLightning/lightning-flash#176] [Tasks]
## ❓ Questions and Help

model.predict is running really slow for image classification. Is there any way to up the number of workers?

![image](https://user-images.githubusercontent.com/3045407/108928077-43146680-764a-11eb-8f48-94cb0506f4cb.png)

**Comment by Borda** [comment]
I would suggest to use `multiprocessing.cpu_count()`
